src/test/regress/sql/multi_name_lengths.sql

```diff
@@ -194,7 +196,8 @@ ORDER BY 1 ASC, 2 ASC
 LIMIT 1 \gset
 
 \di :sneaky_index_name
-SELECT "Constraint", "Definition" FROM table_checks WHERE relid= :sneaky_shard_oid ORDER BY 1 DESC, 2 DESC;
+-- The commit e59fcbd712c777eb2987d7c9ad542a7e817954ec in PostgreSQL 17
+SELECT "Constraint", CASE WHEN "Definition" ~ '^CHECK \\(.*\\)$' THEN regexp_replace("Definition", '^CHECK \\((.*)\\)$', 'CHECK \\1', 'g') ELSE "Definition" END as adjusted_definition FROM table_checks WHERE relid= :sneaky_shard_oid ORDER BY 1 DESC, 2 DESC;
```

Comment:
Why do we need to change this line? Isn't the normalization rule enough?

Reply:
I agree, we should either be doing regexp_replace thing here or should have that normalization rule, not both

Reply:
If not necessary, I vote to definitely use the normalization rule. Let's keep the test readable and hide scary regex lines in the `normalize.sed` file 😆

Reply:
Removed changes other than the sed file.